marklogic-data-hub-glue-connector/src/test/java/com/marklogic/hub/cloud/aws/glue/AbstractGlueConnectorTest.java

```diff
@@ -0,0 +1,63 @@
+package com.marklogic.hub.cloud.aws.glue;
+
+import com.marklogic.hub.HubClient;
+import com.marklogic.hub.impl.HubConfigImpl;
+import com.marklogic.hub.test.AbstractHubTest;
+import org.junit.jupiter.api.BeforeEach;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * Base class for all glue-connector tests. This assumes that we want to run every test as a data-hub-operator user.
+ */
+public class AbstractGlueConnectorTest extends AbstractHubTest {
```

Comment:
This is a base test class to reuse what's in AbstractHubTest. We'll likely add stuff to here to fire up a SparkContext for testing.